mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/okonomi/models/RefusjonskravDto.kt

```diff
@@ -40,4 +41,11 @@ data class RefusjonskravDto(
     data class Tiltakstype(
         val navn: String,
     )
+
+    @Serializable
+    data class Betalingsinformasjon(
+        val kontoNummer: String,
```

Comment:
Angående navngiving så synes jeg `kontonummer` flyter litt bedre enn `kontoNummer/konto_nummer`, men er kanskje litt forvirrende siden vi ikke har vært helt konsekvente når det kommer til orddeling i kodebasen her :see_no_evil: 

En annen ting vi kunne vurdert er å innføre [value classes](https://kotlinlang.org/docs/inline-classes.html) for `kontonummer` og `kid`. Ta en titt på hvordan dette er løst for `Organisasjonsnummer`- eller `NorskIdent`-klassene. Fordelen med slike klasser er at vi får bedre typesikkerhet, selv om begge verdiene i praksis bare er en string.